docs/aerodromes/Brisbane.md

```diff
@@ -38,6 +38,12 @@ BN ADC is responsible for the Class C airspace in the BN CTR `SFC` to `A007`.
   <figcaption>YBBN Maneuvering Area South</figcaption>
 </figure>
 
+## Standard Taxi Directions
+
+Taxiway A is to be used in the same direction as the duty runway. Taxiway B is to be used in the opposite direction to the duty runway.
+
+Taxiway Y is to be used in a westerly direction and Taxiway Z in an easterly direction.
+
```

Comment:
Would be worth including a diagram to supplement this, using the aerodrome or apron chart from the DAPs perhaps

Reply:
I have added a diagram similar to the one found in the Sydney aerodrome page. I have also changed the heading to "Standard Taxi Routes" for consistency with Sydney.